lib/graph3d/DataGroup.js

```diff
@@ -0,0 +1,240 @@
+var DataSet  = require('../DataSet');
+var DataView = require('../DataView');
+var Point3d  = require('./Point3d');
+var Range    = require('./Range');
+
+
+function DataGroup() {
+  this.dataTable = null;  // The original data table
```

Comment:
Mention the consequences for this assignment in the constructor comment. For example, do we need to call `initializeData`? What happens when we don't call `initializeData`?